src/style.css

```diff
@@ -1,3 +1,106 @@
+html {
+  margin: 0;
+  font-family: 'Roboto ', sans-serif;
+  font-size: 12px;
+
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  position: relative;
+  align-items: center;
+  width: 1024px;
+  height: 60px;
+  justify-content: center;
+
+  text-transform: uppercase;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  left: 50px;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+  display: inline-block;
+  align-items: center;
+  justify-content: center;
+}
+
+.ul__nav {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.li__nav {
+  display: flex;
+  margin-right: 20px;
+
+  position: relative;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  text-decoration: none;
+  align-items: center;
+  font-weight: 500;
+}
+
+.li__nav:last-of-type {
+  margin: 0;
+}
+
+.nav__link:link {
+  color: #000;
+}
+
+.nav__link:visited {
+  color: #000;
+}
+
+.nav__link:hover {
+  color: var(--text-color);
+}
+
+.nav__link:active {
+  color: blue;
+}
+
+.nav__link[data-qa='hover']:hover {
+  color: var(--text-color);
+}
+
+.nav__link.is-active {
+  color: var(--text-color);
+  position: relative;
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  right: 0;
+  left: 0;
+  background-color: #00acdc;
+  border-radius: 25%;
+}
+
+@media (width: 1200px) {
+  .container {
+    width: 1200px;
+  }
```

Comment:
Media queries should use 'min-width' or 'max-width' rather than 'width' for better responsiveness across different screen sizes.